test/features/install_explainability.feature

```diff
@@ -0,0 +1,56 @@
+@explainability
+@kafka
+Feature: Kogito Explainability service
+
+  Background:
+    Given Namespace is created
+
+  @smoke
+  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
```

Comment:
As far I can see, the https://issues.redhat.com/browse/KOGITO-3176 ticket has two PRs and both merged, maybe this is already fixed?

Reply:
Hi @Sgitario , those PR are just disabling this test. They are not the fix. I'll remove them from the ticket